utils.py

```diff
@@ -30,3 +31,60 @@ def center_of_tile(x: int, y: int) -> Vector:
         ((x // TILE_SIZE) * TILE_SIZE) + TILE_SIZE // 2,
         ((y // TILE_SIZE) * TILE_SIZE) + TILE_SIZE // 2,
     )
+
+
+def process_objects(file_path: str) -> list:
```

Comment:
the only criticism i have is for more accurate typehints, e.g. `list[dict[... , ...]]`